velox/type/UnscaledLongDecimal.h

```diff
@@ -161,6 +161,21 @@ struct UnscaledLongDecimal {
     return *this;
   }
 
+  static inline void serialize(
+      char* serializedData,
+      const UnscaledLongDecimal& longDecimal) {
+    *reinterpret_cast<uint64_t*>(serializedData) =
+        (uint64_t)longDecimal.unscaledValue();
+    *reinterpret_cast<uint64_t*>(serializedData + sizeof(uint64_t)) =
+        ((int64_t)(longDecimal.unscaledValue() >> 64));
```

Comment:
Why we cast the lower part to int64_t but not uint64_t?

Reply:
Fixed it. Actually doesn't matter. While deserializing the data we cast appropriately.